planner/core/integration_test.go

```diff
@@ -8431,3 +8431,93 @@ func TestIssue40285(t *testing.T) {
 	tk.MustExec("CREATE TABLE t(col1 enum('p5', '9a33x') NOT NULL DEFAULT 'p5',col2 tinyblob DEFAULT NULL) ENGINE = InnoDB DEFAULT CHARSET = latin1 COLLATE = latin1_bin;")
 	tk.MustQuery("(select last_value(col1) over () as r0 from t) union all (select col2 as r0 from t);")
 }
+
+func TestIsIPv4ToTiFlash(t *testing.T) {
```

Comment:
unit test in `expression/expr_to_pb_test.go` need to be added too, like https://github.com/pingcap/tidb/pull/37589/files#diff-6d884fa56c800263f08f60430b9a757c8c536ebfe98de09ec031f89680147765R1181-R1185

Reply:
done,thx~